### PR TITLE
Add data model

### PR DESCRIPTION
**Pull request by drewsef-916**

### CODE REVIEW

**Comment by snsmith92** [comment]
@drewsef-916 are these changes ready to be merged to master?

**Comment by ilyakrasnov** [comment]
Looks great @drewsef-916 👍

File: app/models/game.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 class Game < ApplicationRecord
+  scope :available, -> { where(Game.arel_table[:white_player].not_eq(0)).where(black_player: nil) }
```
Comment (ilyakrasnov) [inline app/models/game.rb]: `white_player` should be refactored to `owner` and `black_player` to `opponent`

File: db/migrate/20180205144520_add_black_white_players_to_games.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class AddBlackWhitePlayersToGames < ActiveRecord::Migration[5.0]
+  def change
+    add_column :games, :black_player, :integer
+    add_column :games, :white_player, :integer
```
Comment (ilyakrasnov) [inline db/migrate/20180205144520_add_black_white_players_to_games.rb]: `black` and `white` player as in commend above :)

**Comment by ilyakrasnov** [comment]
@drewsef-916 @snsmith92 should we then close this pr?

**Comment by snsmith92** [comment]
I'm just going to close this pull request since we've committed add_data_model_2

**Pull request closed without merging**